spec/models/ingredient_mention_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe IngredientMention, type: :model do
+  subject do
+    build(:ingredient_mention)
+  end
+
+  describe 'when observing relationships' do
+    context 'with movies' do
+      it 'belongs to an ingredient' do
```

Comment:
These seem flipped. Test the movie in the movie context and the ingredient in the ingredient context.